pubspec.yaml

```diff
@@ -12,7 +12,7 @@ environment:
   sdk: '>=2.6.0 <3.0.0'
 
 dependencies:
-  analyzer: ^0.39.3
+  analyzer: ^0.39.6
```

Comment:
Analyzer 0.39.6 has a few new classes in the Mock SDK needed for testing these changes.